Rewrite landing navigation (Getting Started) (#1217)
prysmaticlabs/documentation: docs/learn/dev-concepts/prysm-validator-client.md

<HeaderBadgesWidget />

> Although [beacon nodes](/learn/dev-concepts/prysm-beacon-node.md) handle network synchronization, drawing consensus and performing several other low-level functions, the role of [validators](/terminology#validator) whom stake `ETH` to in order to perform block [proposals](/terminology#proposal-propose-) and [attestations](/terminology#attestation-attest-) are an equally critical component of the Ethereum beacon chain.

Reviewer: ```suggestion
Although [beacon nodes](http://learn/dev-concepts/prysm-beacon-node.md) handle network synchronization, draw consensus, and perform several other low-level functions, the role of [validators](http://terminology/#validator) who stake `ETH` to propose blocks and attest is equally critical to the Ethereum beacon chain.
```

As mentioned, validators have two responsibilities: to [propose](/terminology#proposal-propose-) (or produce) blocks known as beacon blocks, which contain consensus information about shards across the network, or to [attest](/terminology#attestation-attest-) (or vote on) the validity of blocks that have already been produced.

![Validator](/images/prysm-validator.png)

## How does it work?

> A validator instance is permitted to begin participating in the network once 32 `ETH` is locked up in a [validator deposit contract](/learn/dev-concepts/validator-deposit-contract.md). Validators are tasked with correctly [proposing](/terminology#proposal-propose-) or [attesting](/terminology#attestation-attest-) to blocks on the beacon chain, and receive either rewards or penalties to the initial deposit based upon their overall performance.

Reviewer: ```suggestion
A validator instance is permitted to begin participating in the network once 32 `ETH` has been locked in a [validator deposit contract](http://learn/dev-concepts/validator-deposit-contract.md). Validators are tasked with correctly [proposing](http://terminology/#proposal-propose-) or [attesting](http://terminology/#attestation-attest-) to blocks on the beacon chain and receive rewards or penalties on their initial deposit based on their overall performance.
```

1. Waits for the event log signaling a start to have occurred in the [validator deposit contract](/learn/dev-concepts/validator-deposit-contract.md).
2. Checks if public key corresponding to the validator instance has been activated on the beacon chain.
3. A validator is assigned to a shard either as a[ proposer](/terminology#proposal-propose-) \(creator\) or [attester](/terminology#attestation-attest-) (voter).

> 4. The validator then has a ticker that works every slot (6 seconds). If the slot ticks at the validator's assigned slot, a beacon block is either [proposed](/terminology#proposal-propose-) or [attested](/terminology#attestation-attest-), depending on assigned role.

Reviewer: ```suggestion
4. The validator then has a ticker that works every slot (6 seconds). If the slot ticks at the validator's assigned slot, a beacon block is either [proposed](/terminology#proposal-propose-) or [attested](/terminology#attestation-attest-), depending on the assigned role.
```
